pkgs/build-support/compress-drv/default.nix

```diff
@@ -13,6 +13,10 @@
 
   : List of file extensions to compress. Example: `["txt" "svg" "xml"]`.
 
+  `extraFindOperands` (String)
+
+  : Exclude pattern. Example: `-not -iregex ".*(\/apps\/.*\/l10n\/).*"`
```

Comment:
```suggestion
  : Extra command line parameters to pass to the [`find`](https://en.wikipedia.org/wiki/Find_(Unix)) command. This might be useful for excluding files. For example: `-not -iregex ".*(\/apps\/.*\/l10n\/).*"`
```

Reply:
Not going to link wikipedia here, that won't help people.